custom-connectors/PUGGamifiedEngagement/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PICNIC CONNECTOR",
```

Comment:
@smart-jp Your connector's folder name is `PUGGamifiedEngagement` and but your connector title is mentioned as `PICNIC CONNECTOR`. Can you please let us why is like that. We expect the connector's title and folder name should be same.

And the connector's title doesn't meet the standards. Can you please make sure that you are following all the instructions mentioned in the [documentation](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#submission-requirements) to get certify your connector.

Reply:
@smart-jp The title should not contain `CONNECTOR` word and please remove all capitalization.